README.md

```diff
@@ -0,0 +1,23 @@
+# NI JavaScript Style Guide
+
+Welcome to NI's internal and external JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started) and [TSLint](https://palantir.github.io/tslint/).
+
+## Installation
+
+```
+npm install --save-dev eslint eslint-plugin-import @ni/eslint-config
+```
+
+## Usage
+
+Extend @ni in the ESLint configuration.
+
+```json
+{
+    "extends": "@ni"
+}
+```
+
+## License
+
+[MIT (c) 2020 NI](./LICENSE)
```

Comment:
I think for legal stuff we are supposed to use [National Instruments Corporation](https://teams.microsoft.com/l/message/19:c0ff86b3253946e1948fe7089cd0bc91@thread.skype/1594757999233?tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e&groupId=35260d9f-7a49-4099-b8cc-13346dcbab4e&parentMessageId=1594743825112&teamName=Open%20Source&channelName=General&createdTime=1594757999233)